config/conversations/questions/whats_for_lunch.rb

```diff
@@ -0,0 +1,99 @@
+require "net/https"
+
+Houston::Slack.config do
+  listen_for(/what.*s for lunch(?:\s(?:on\s)?(?<modifier>next\s)?(?<day>.*))?\??/i) do |e|
+    day = e.match[:day] || "today"
+    day = day.downcase.gsub(/\?/, "")
+    day = "today" unless day =~ /^(?:mon|tues|wed(?:nes)?|thu(?:rs)?|fri)(?:day)?$|^tomorrow$/
+
+    target_date = Date.today
+    target_date += 1.day if day == "tomorrow"
+    unless day =~ /today|tomorrow/
+      find_next_week = !e.match[:modifier].nil?
+      target_wday = case day
+      when /mon/
+        1
+      when /tue/
+        2
+      when /wed/
+        3
+      when /thu/
+        4
+      when /fri/
+        5
+      end
+      today_wday = target_date.wday
+      if today_wday > target_wday
+        day_diff = 7 - (today_wday - target_wday)
+        target_date += day_diff.days
+      elsif today_wday == target_wday && find_next_week
+        target_date += 7.days
+      else
+        day_diff = target_wday - today_wday
+        target_date += day_diff.days
+      end
+    end
+
+    e.reply "Hold on, I'll check..."
```

Comment:
I think you can drop _Hold on, I'll check...: since the response comes in less than a second, Houston doesn't need to prepare it's conversation-buddy for a wait.